src/scripts/global-data.lua

```diff
@@ -54,24 +54,76 @@ local function build_amount_string(material)
     amount_string = (probability * 100).."% "..amount_string
   end
 
-  -- temperature
-  local temperature = material.temperature
-  local temp_min = material.minimum_temperature
-  local temp_max = material.maximum_temperature
-  if temperature then
-    amount_string = amount_string.."  ("..math.round_to(temperature, 2).."°)"
-  elseif temp_min and temp_max then
-    if temp_min == -0X1.FFFFFFFFFFFFFP+1023 then
-      amount_string = amount_string.."  (<= "..math.round_to(temp_max, 2).."°)"
-    elseif temp_max == 0X1.FFFFFFFFFFFFFP+1023 then
-      amount_string = amount_string.."  (>= "..math.round_to(temp_min, 2).."°)"
-    else
-      amount_string = amount_string.."  ("..math.round_to(temp_min, 2).." - "..math.round_to(temp_max, 2).."°)"
+  -- second return is the "average" amount
+  return amount_string, amount == nil and ((material.amount_min + material.amount_max) / 2) or nil
+end
+
+local function build_temperature_strings(temperatureMin, temperatureMax)
+  if temperatureMin == temperatureMax then
```

Comment:
Please use `camel_case` everywhere. Also, please re-add the `math.round_to(temp, 2)`s.